spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/SparkExpressionConverter.scala

```diff
@@ -41,6 +41,7 @@ trait SparkExpressionConverter {
       .orElse(convertBooleanExpressions(expression, fields))
       .orElse(convertMiscExpressions(expression, fields))
       .orElse(convertStringExpressions(expression, fields))
+      .orElse(convertMathematicalExpressions(expression, fields))
```

Comment:
Minor: Can you move this to above `convertMiscExpressions` to preserve alphabetical order?

Reply:
Yeah Sure